src/main/webapp/definitions/alfresco-content-models.yaml

```diff
@@ -0,0 +1,738 @@
+        * name
+        * namespaceUri
+        * namespacePrefix 
+      operationId: listAspectProperties
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/modelIdParam'
+        - $ref: '#/parameters/aspectNameParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+        - $ref: '#/parameters/propertyEntryIncludeParam'
+        - $ref: '#/parameters/orderByParam'
+      responses:
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/PropertyPaging'
+        '400':
+          description: |
+            Invalid parameter: value of **maxItems**, **skipCount**, **orderBy**, **modelId** or **aspectName** is invalid
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to retrieve **modelId**
+        '404':
+          description: |
+            **modelId** or **aspectName** does not exist
+  '/models/{modelId}/constraints':
+    get:
+      x-alfresco-since: "6.2.2"
+      tags:
+        - models
+      summary: List constraints for the model
+      description: |
+        **Note:** this endpoint is available in Alfresco 6.2.2 and newer versions.
+
+        Get the list of constraints defined in the model **modelId**.
+
+        The default sort order is **name** ascending, but you can use an optional **ASC** or **DESC** 
+        modifier to specify an ascending or descending sort order. 
+
+        For example, specifying ```orderBy=name DESC``` returns QName entries in descending **name** order.
+        You can specify one or more of the following fields in the **orderBy** parameter:
+        * name
+        * namespaceUri
+        * namespacePrefix 
+      operationId: listCostraints
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/modelIdParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+        - $ref: '#/parameters/orderByParam'
+      responses:
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/QNamePaging'
+        '400':
+          description: |
+            Invalid parameter: value of **maxItems**, **skipCount**, **orderBy** or **modelId** is invalid
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to retrieve **modelId**
+        '404':
+          description: |
+            **modelId** does not exist
+  '/models/{modelId}/constraints/{constraintName}':
```

Comment:
You've probably guessed by now.... please use constraintId here instead of constraintName for consistency